doc/interceptors.md

```diff
@@ -0,0 +1,61 @@
+## HTTP Interceptors
+
+With the release of sync-android version TBD a new HTTP Interceptor API was introduced.
+HTTP Interceptors allow the developer to modify the HTTP Requests and Responses during
+the replication process.
+
+In order to make changes to a HTTP request, you need to implement the interface
+`HTTPConnectionRequestInterceptor`, for HTTP responses, you implement the interface
+`HTTPConnectionResponseInterceptor`. A Class may implement both interface without causing
+problems, an example of this is the `CookieInterceptor` class.
+
+In order to add an Http Interceptor to a replication, you call the `addResponseInterceptors` or
```

Comment:
Suggest always referring to request/response in the same order, so here you'd swap the `addResponseInterceptors` and `addRequestInterceptors` method names.